pkg/platform/aws/aws.go

```diff
@@ -140,7 +140,7 @@ func (c *config) Apply(ex *terraform.Executor) error {
 		return err
 	}
 
-	return ex.Apply(nil)
+	return ex.Apply([]string{"-parallelism=100"})
```

Comment:
Maybe this can be part of executor default arguments? Or maybe at least we can put it into a function, so we de-duplicate the code?

Reply:
Also, why 100 is a good number? Doesn't that overwhelm Terraform/CPU/RAM?

Reply:
> Maybe this can be part of executor default arguments? Or maybe at least we can put it into a function, so we de-duplicate the code?

@invidian 

I think its not big of a deal to add extra arguments this way, this is also the same for where we pass `-parallelism=1` for terraform execution without parallelism.

One thing we can do to improve is to to convernt `--parallelism=X` as a const so that we we don't have to change everywhere if there is ever a value change for `X`.

Reply:
10 meant that with 30 resources to create it took 3 times as long as needed. I just chose 100 arbitrarily with the assumption that people have less resources to create even for a very large cluster. Having 100 processes at maximum should not be an issue for normal user machines as most of these processes can share pages and don't use the CPU much but rather have to wait for an API call to finish.

Reply:
I think having it as a constant and documented, that it's an arbitrary number is acceptable. In the long term, this sort of logic should be platform-independent (I'm sure there is an issue for this).